<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="settingsTab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="currentDataTab">
     <attribute name="title">
      <string>Current Data</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="currentDateTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temperatureLabel">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="settingsTab">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="maxTemperatureLabel">
          <property name="text">
           <string>Max Temperature</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="maxTemperatureSpinBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="minTemperatureLabel">
          <property name="text">
           <string>Min Temperature</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="minTemperatureSpinBox"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QQuickWidget" name="quickWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="resizeMode">
           <enum>QQuickWidget::SizeRootObjectToView</enum>
          </property>
          <property name="source">
           <url>
            <string>qrc:/inputpanelarea.qml</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="historyTab">
     <attribute name="title">
      <string>History</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="temperaturehistoryform" name="historyForm" native="true"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>761</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="heatingInd">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">*:enabled {
	background-color: rgb(255, 211, 212);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string>Heating</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="coolingInd">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">*:enabled {
	background-color: rgb(170, 255, 255);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string>Cooling</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fanInd">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">*:enabled {
	background-color: rgb(255, 255, 171);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string>Fan</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMainWindow">
    <property name="title">
     <string>MainWindow</string>
    </property>
   </widget>
   <addaction name="menuMainWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>temperaturehistoryform</class>
   <extends>QWidget</extends>
   <header>temperaturehistoryform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
